core/types/suave_structs.go

```diff
@@ -22,6 +25,17 @@ type BuildBlockArgs struct {
 	FillPending    bool
 }
 
+type buildBlockArgsMarshaling struct {
```

Comment:
The types that belong to suavex are handled now on suave-execution-geth. I already applied this fix there too https://github.com/flashbots/suave-execution-geth/pull/13